app/style/ReactSelectGlobal.css

```diff
@@ -226,7 +226,6 @@
   height: 0;
   width: 0;
   position: relative;
-  transition: all var(--input-transition-time) var(--input-timing-function);
```

Comment:
are these used in other places 🤔 ?

Reply:
@amassarwi Thanks for the review. 
This transition only affects the arrow icon. I've removed the line for the sake of uniformity. Other similar arrows aren't animated in the app.

Reply:
i mean the theme variables: `input-transition-time` & `input-timing-function` - i'm not sure if they are used/needed in other places, if not we could ditch them from the theme files.

Reply:
Ahh, I see :) They are used in this file, but ditching them hasn't changed the behavior of the select control.

Reply:
Noice, thanks 🥇